modules/data.land/R/extract_SM_CDS.R

```diff
@@ -0,0 +1,82 @@
+#' Extract CDS soil moisture data for the SDA workflow.
+#'
+#' @param site_info Bety list of site info including site_id, lon, and lat.
+#' @param time.points A vector contains each time point within the start and end date.
+#' @param in.path physical paths to where the unziped soil moisture files are downloaded.
+#' @param out.path Where the final CSV file will be stored.
+#' @param allow.download Flag determine if we want to automatic download files if they are not available.
+#' @param search_window search window for locate available soil moisture values.
```

Comment:
not clear what units this is in. pixels? km? degrees? if pixels, include a reminder of how big a pixel is for this data product.

Reply:
It's the time search window (in days). Because it will take ~ 10 days for this product to get a global coverage, we need to account for that to grab sufficient estimations for all locations that we are interested in.

Reply:
Updated the documentation to make it more clear.